Lib/test/test_cgi.py

```diff
@@ -129,6 +129,20 @@ def test_parse_multipart(self):
                     'file': [b'Testing 123.\n'], 'title': ['']}
         self.assertEqual(result, expected)
 
+    def test_parse_multipart_without_content_legth(self):
```

Comment:
Hi, here's a typo => test_parse_multipart_without_content_legth should be test_parse_multipart_without_content_length (missing "n")